### ISSUE TITLE
question/effort request: implementing Switcheroo as a part of Microsoft's PowerToy's PowerRun

### ISSUE THREAD

**Issue by AdamJel** [kvakulo/Switcheroo#157]
Hi,

Switcheroo is really great in its category. For me, there is no match for it. My current setup is [PowerRun](https://github.com/microsoft/PowerToys) and Switcheroo. PowerRun actually has switching function of running processes, but it does not work as well as Switcheroo. Could it be possible to get in touch witch that functionality developers and talk about migration of Switcheroo into PowerRun?

Having Switcheroo in PowerRun would be the single best user experience. [Wox](https://github.com/Wox-launcher/Wox) already did this move and I believe all users benefit from it.

What do you think @bao-qian, @crutkas?

**Comment by jsonMartin** [comment]
I agree that Switcheroo is a way more efficient task switcher than **PowerToys Run**. I use **PowerRun** also as a launcher and for doing simple math problems etc, but I find it severely lacking as an Alt-Tab replacement

I like Switcheroo so much that I ended up making my own repo that has all the combined extra features I could find from other forks, such as Dark Mode/VIM key navigation/etc, and consolidated it under one codebase: https://github.com/jsonmartin/switcheroo
